scripts/util.py

```diff
@@ -152,6 +155,61 @@
     None
 ] = lambda self, data: yaml.representer.SafeRepresenter.represent_str(self, str(data))
 
+with open(PROVIDERS_FILE) as stream:
+    try:
+        providers = yaml.safe_load(stream)
+    except yaml.YAMLError as exc:
+        providers = None
+        print(exc)
+prv = NSDict(providers)
+
+universe_domain = "googleapis.com"
+universe_sa_credentials = None
+endpoint_versions = None
+if prv.universe_information:
+    d, c = "domain", "sa_credentials"
+    if d in prv.universe_information and prv.universe_information[d]:
+        universe_domain = prv.universe_information[d]
+        # Only use credentials if domain is different
+        if c in prv.universe_information and prv.universe_information[c]:
+            universe_sa_credentials = "universe_sa_credentials.json"
```

Comment:
Where does this file comes from?

Reply:
No longer using credentials in here.  They're supposed to be SA creds so that should be handled by what's in here already.